.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
 
   rust:
     name: Rust
-    runs-on: ubuntu-latest
+    runs-on: windows-latest
```

Comment:
Changed to Windows to ensure the Windows TTS code compiles fine.

Reply:
Sounds good to me.